app/models/info_request/prominence/embargo_expired_today_query.rb

```diff
@@ -0,0 +1,23 @@
+# -*- encoding : utf-8 -*-
+class InfoRequest
+  module Prominence
+    class EmbargoExpiredTodayQuery
+      def initialize(relation = InfoRequest)
+        @relation = relation
+      end
+
+      def call
+        @relation.
+          includes(:embargo).
+          joins(:info_request_events).
+          where('embargoes.info_request_id IS NULL').
+          where([
+            'info_request_events.created_at >= ?',
+            Time.zone.now.beginning_of_day
+          ]).
+          where('info_request_events.event_type' => 'expire_embargo').
```

Comment:
can't be `where('info_request_events.event_type': 'expire_embargo').` below Ruby 2.3, see: https://travis-ci.org/mysociety/alaveteli/builds/289446372

Reply:
I think you can do `where(info_request_events: { event_type: 'expire_embargo' })`

Reply:
ah, yes - that looks better, cheers!